packages/react-generator/src/generator/page.js

```
@@ -376,34 +383,82 @@ const generateReactCode = ({ schema, name, type, componentsMap }) => {
+  const stringUseMemo = `
+    useMemo(() => {
+      ${shouldComponentUpdate && shouldComponentUpdate['body'] ? shouldComponentUpdate['body'] : ''}
+      }, [${shouldComponentUpdate && shouldComponentUpdate['params'] ? shouldComponentUpdate['params'] : ''}])
+  `
   const { imports } = generateReactImports(description, name, type, componentsMap)
 
+  console.log(getters.join('\n'), 'current>>>>>>')
+
+  console.log(arrowMethods.join('\n'), 'arrowMethod>>>>>>')
+
+  console.log(statement, 'stateStatement>>>>>>')
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)